src/app/public/modules/list-view-grid/list-view-grid.component.spec.ts

```diff
@@ -537,110 +537,97 @@ describe('List View Grid Component', () => {
         fixture.detectChanges();
         tick();
         fixture.detectChanges();
-        expect(fixture.nativeElement.querySelector('.sky-grid-row-delete-heading')).toBeNull();
+        expect(document.querySelector('#row-delete-ref-1')).toBeNull();
```

Comment:
All these changes to the spec file were to account for a change from this PR: https://github.com/blackbaud/skyux-grids/pull/157/files.

They are unrelated to this PR.